src/storage_fuzz.zig

```diff
@@ -0,0 +1,164 @@
+const std = @import("std");
+const assert = std.debug.assert;
+const log = std.log.scoped(.fuzz_storage);
+
+const stdx = @import("stdx.zig");
+const vsr = @import("vsr.zig");
+const constants = @import("constants.zig");
+const IO = @import("testing/io.zig").IO;
+const Storage = @import("storage.zig").Storage(IO);
+const fuzz = @import("testing/fuzz.zig");
+
+pub fn main(args: fuzz.FuzzArgs) !void {
+    const zones: []const vsr.Zone = &.{
+        .superblock,
+        .wal_headers,
+        .wal_prepares,
+        .client_replies,
+    };
+
+    const sector_count = 64;
+    const storage_size = sector_count * constants.sector_size;
+    const iterations = args.events_max orelse 10_000;
+
+    const failed_sector_cluster_count = 7;
+    const failed_sector_cluster_minimum_length = 1;
+    const failed_sector_cluster_maximum_length = 4;
```

Comment:
We can also generate these randomly (along with sector count) from within a range, to maximize our fuzz surface area!